src/renderer/components/+config-autoscalers/hpa-v1-metric-parser.injectable.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { HorizontalPodAutoscalerV1MetricParser } from "./hpa-v1-metric-parser";
+
+const horizonalPodAutoscalerV1MetricParser = getInjectable({
+  id: "horizontal-pod-autoscaler-v1-metric-parser",
+  instantiate: () => {
+    return new HorizontalPodAutoscalerV1MetricParser();
```

Comment:
Why do these need to be injectables? All the methods look pure (as in no side effects) and have no dependencies either.

Reply:
Moved away from injectable form.